code/datums/elements/food/processable.dm

```diff
@@ -0,0 +1,33 @@
+// If an item has the processable item, it can be processed into another item with a specific tool. This adds generic behavior for those actions to make it easier to set-up generically.
+/datum/element/processable
+	element_flags = ELEMENT_BESPOKE
+	id_arg_index = 2
+	///The result item
+	var/atom/result
```

Comment:
Is this a path? An instantiated entity? Document it properly and give it a proper name.

Reply:
sure